manifests/params.pp

```diff
@@ -23,6 +23,7 @@
         }
       }
       $config_dir             = '/etc/libvirt'
+      $default_dir            = '/etc/default'
```

Comment:
can you change this to the path of the file, I do not think that we ever need another file in this directory.